### DISCUSSION TITLE
Provide non-touch configuration

### DISCUSSION THREAD

**Discussion by andremalenfant** [KlipperScreen/KlipperScreen#478] [Ideas]
It would be nice to be able to specify that the screen has no touch capability and remove all menus/buttons to use all screen real estate for monitoring only.

**Comment by alfrix** [comment]
i'm sorry but this project was created from the start to be a touchscreen GUI, in it's incompatible with this idea

**Reply by andremalenfant** [reply]
I understand that it was the original intention, but it would also be useful to be used as a way to display the status on a wall display.
